app/index.js

```diff
@@ -92,8 +96,17 @@ class HomeButler {
         const temperature = cardDOM.getElementById('temperature')
         const humidity = cardDOM.getElementById('humidity')
 
-        temperature.text = device.attributes.current_temperature + '°'
-        humidity.text = device.attributes.current_humidity + '%'
+        temperature.text =
+          device.attributes.current_temperature !== undefined &&
+          device.attributes.current_temperature !== null
+            ? device.attributes.current_temperature + "°"
+            : "";
+
+        humidity.text =
+          device.attributes.current_humidity !== undefined &&
+          device.attributes.current_humidity !== undefined
```

Comment:
Should be `!== null`? Might be better to use isNaN()

Reply:
Thanks for the minor fixes. Especially this was a copy paste mistake... sorry about that.

Please keep in mind that `isNaN(undefined) === true` as expected, but `isNaN(null) === false`. It might be possible that `null%` will be shown. 

However, this shouldn't happen very often. In my case the reason is that I'm using a custom_component to get the current temperature from other sensors into my climates devices.

But since neither `null°` nor `null%` lead to app crashes, the current behavior should be fine for my rare case as well 👍